library/src/main/kotlin/org/codice/compliance/SAMLSpecRefMessage.kt

```diff
@@ -205,197 +205,197 @@ enum class SAMLSpecRefMessage(val message: String) {
 
-    SAMLBindings_3_5_4_d1("The action attribute of the form MUST be the recipient's HTTP endpoint for the " +
-            "protocol or profile using this binding to which the SAML message is to be delivered."),
+    SAMLBindings_3_5_4_d1("""The action attribute of the form MUST be the recipient's HTTP endpoint for the " +
+            "protocol or profile using this binding to which the SAML message is to be delivered."""),
 
-    SAMLBindings_3_5_4_d2("The method attribute [of the form] MUST be \"POST\"."),
+    SAMLBindings_3_5_4_d2("""The method attribute [of the form] MUST be "POST"."""),
 
-//    todo SAMLBindings_3_5_4_c("SAML message could not be decoded. SAML messages should be base-64 encoded."),
+//    todo SAMLBindings_3_5_4_c("""SAML message could not be decoded. SAML messages should be base-64 encoded."""),
 
-    SAMLBindings_3_5_5_2_a("If the message is signed, the Destination XML attribute in the root SAML element of the " +
-            "protocol message MUST contain the URL to which the sender has instructed the user agent to deliver the " +
-            "message."),
+    SAMLBindings_3_5_5_2_a("""If the message is signed, the Destination XML attribute in the root SAML element of the """ +
+            """protocol message MUST contain the URL to which the sender has instructed the user agent to deliver the """ +
+            """message."""),
 
-    SAMLBindings_3_5_6_a("HTTP interactions during the message exchange MUST NOT use HTTP error status codes " +
+    SAMLBindings_3_5_6_a("""HTTP interactions during the message exchange MUST NOT use HTTP error status codes " +
             "to indicate failures in SAML processing, since the user agent is not a full party to the SAML protocol " +
-            "exchange."),
+            "exchange."""),
 
     /***************
      *
      * XML Datatype Schema
      *
      ***************/
 
-    XMLDatatypesSchema_3_2_7("See XML Schema 2 [https://www.w3.org/TR/xmlschema-2/] section 3.2.7 for more " +
-            "information"),
+    XMLDatatypesSchema_3_2_7("""See XML Schema 2 [https://www.w3.org/TR/xmlschema-2/] section 3.2.7 for more " +
+            "information"""),
 
-    XMLDatatypesSchema_3_2_7_1_a("if [the year has] more than four digits, leading zeros are prohibited"),
+    XMLDatatypesSchema_3_2_7_1_a("""if [the year has] more than four digits, leading zeros are prohibited"""),
 
-    XMLDatatypesSchema_3_2_7_1_b("'0000' is prohibited [as the year]"),
+    XMLDatatypesSchema_3_2_7_1_b("""'0000' is prohibited [as the year]"""),
 
-    XMLDatatypesSchema_3_2_7_1_c("a plus sign is not permitted [in the year]"),
+    XMLDatatypesSchema_3_2_7_1_c("""a plus sign is not permitted [in the year]"""),
 
     /***************
      *
      * XML Signature Syntax and Processing
      *
      ***************/
 
-    XMLSignature_4_5("compliant versions [of KeyInfo] MUST implement KeyValue")
+    XMLSignature_4_5("""compliant versions [of KeyInfo] MUST implement KeyValue""")
 }
```

Comment:
❓ Do you not need a new line at the end?

Reply:
I do. It's odd on github.